### PR TITLE
{bio}[gompi/2019a] BLASR v5.3.3, pbcopper v1.3.0, pbbam v1.0.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#8926~~ (cram), ~~#8927~~ (gtest)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c385faf6b48bb3cf57181e52af2e68de for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3121.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c2241d5b291607fa2faef97c7072f1e5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/e59ea1f5d4479f65591652cd3230c7f2 for a full test report.

File: easybuild/easyconfigs/b/BLASR/BLASR-5.3.3-gompi-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'cstd': 'c++14'}
+
+source_urls = ['https://github.com/PacificBiosciences/blasr/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['0c69f0ed04c6998fdd60969dc6c87f29298453a230767f5f206ccceca939dc52']
+
+builddependencies = [
+    ('Meson', '0.50.0', '-Python-3.7.2'),
```
Comment (smoors) [inline easybuild/easyconfigs/b/BLASR/BLASR-5.3.3-gompi-2019a.eb]: @boegel even though this is Python3 only, is it ok to **not** have a Python versionsuffix for BLASR?

Comment (boegel) [reply easybuild/easyconfigs/b/BLASR/BLASR-5.3.3-gompi-2019a.eb]: @smoors Yes in this case, because `Meson` is just a build dependency...

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
node321.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/922224af4fd506f09146c6d28f5b248b for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**